jdaviz/configs/imviz/helper.py

```diff
@@ -177,14 +187,168 @@ def offset_to(self, dx, dy, skycoord_offset=False):
                     new_sky_cen = sky_cen.spherical_offsets_by(dx, dy)
                 self.center_on(new_sky_cen)
             else:
-                raise AttributeError(f'{image.label} does not have a valid WCS')
+                raise AttributeError(f'{getattr(image, "label", None)} does not have a valid WCS')
         else:
             with delay_callback(viewer.state, 'x_min', 'x_max', 'y_min', 'y_max'):
                 viewer.state.x_min += dx
                 viewer.state.y_min += dy
                 viewer.state.x_max = viewer.state.x_min + width
                 viewer.state.y_max = viewer.state.y_min + height
 
+    @property
+    def marker(self):
+        """Marker to use.
+
+        Marker can be set as follows; e.g.::
+
+            {'color': 'red', 'alpha': 1.0, 'markersize': 3}
+            {'color': '#ff0000', 'alpha': 0.5, 'markersize': 10}
+            {'color': (1, 0, 0)}
+
+        Also see: https://docs.glueviz.org/en/stable/api/glue.core.visual.VisualAttributes.html
+
+        """
+        return self._marker_dict
+
+    @marker.setter
+    def marker(self, val):
+        # Validation: Ideally Glue should do this but we have to due to
+        # https://github.com/glue-viz/glue/issues/2203
+        given = set(val.keys())
+        allowed = set(('color', 'alpha', 'markersize'))
+        if not given.issubset(allowed):
+            raise KeyError(f'Invalid attribute(s): {given - allowed}')
+        if 'color' in val:
+            from matplotlib.colors import ColorConverter
+            ColorConverter().to_rgb(val['color'])  # ValueError: Invalid RGBA argument
+        if 'alpha' in val:
+            alpha = val['alpha']
+            if not isinstance(alpha, (int, float)) or alpha < 0 or alpha > 1:
+                raise ValueError(f'Invalid alpha: {alpha}')
+        if 'markersize' in val:
+            size = val['markersize']
+            if not isinstance(size, (int, float)):
+                raise ValueError(f'Invalid marker size: {size}')
+
+        # Only set this once we have successfully validated a marker.
+        # Those not set here use Glue defaults.
+        self._marker_dict = val
+
+    def _validate_marker_name(self, marker_name):
+        """Raise an error if the marker_name is not allowed."""
+        if marker_name in RESERVED_MARKER_SET_NAMES:
+            raise ValueError(
+                f"The marker name {marker_name} is not allowed. Any name is "
+                f"allowed except these: {', '.join(RESERVED_MARKER_SET_NAMES)}")
+
+    def add_markers(self, table, x_colname='x', y_colname='y',
+                    skycoord_colname='coord', use_skycoord=False,
+                    marker_name=None):
```

Comment:
I think this should get an extra keyword `marker_style`, which in the current implementation would basically just be:
```
if marker_style is not None:
    self.marker = marker_style
```
at the top of this function.  See my comment above about the `marker` property for the motivation.

Reply:
Out-of-band, @pllim talked me out of this idea.  Instead, a clear note in the docstring saying something like ""wondering how to set the style?  see self.markers" is the way to go so as not to deviate too far from astrowidgets.

Reply:
Yes, we have agreed to pursue this over at astropy/astrowidgets#141 first and open follow-up PR here if necessary.